admin_guide/service_accounts.adoc

```diff
@@ -108,6 +108,9 @@ Users that are members of the *dedicated-admins* group, and thus have been
 granted the *dedicated-admin* role, have `edit` access to the *dedicated-admin*
 project. This allows these users to manage the service accounts in this project
 and create new ones as needed.
+
+Users with a *dedicated-reader* role are granted edit and view access to the
+*dedicated-readers* project and view-only access to the other projects.
```

Comment:
There is a tiny point need to be updated. The project is `dedicated-reader`, but not dedicated-reader**s**.